x-pack/plugins/security_solution/public/overview/pages/landing.tsx

```diff
@@ -5,15 +5,23 @@
  * 2.0.
  */
 
-import React, { memo } from 'react';
+import React, { memo, useMemo } from 'react';
 import { SpyRoute } from '../../common/utils/route/spy_routes';
 import { SecurityPageName } from '../../../common/constants';
 import { LandingPageComponent } from '../../common/components/landing_page';
+import { useKibana } from '../../common/lib/kibana';
+import { PluginTemplateWrapper } from '../../common/components/plugin_template_wrapper';
 
 export const LandingPage = memo(() => {
+  const { getStartedComponent } = useKibana().services;
+  const GetStartedComponent = useMemo(() => getStartedComponent?.(), [getStartedComponent]);
```

Comment:
Won't we always have the `getStartedComponent` defined in the services?

Reply:
So it could be undefined in https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/types.ts#L135